src/test/ui/huge-array-simple.stderr

```diff
@@ -1,4 +1,8 @@
 error: the type `[u8; N]` is too big for the current architecture
+  --> $DIR/huge-array-simple.rs:12:9
+   |
+LL |     let _fat : [u8; (1<<61)+(1<<31)] =
```

Comment:
That span isn't ideal, but it's hard to do better in the MIR, especially as there may be no HIR local in the case of temporaries. Not sure what to do about it

Reply:
I agree, but at the same time it's a huge improvement over the current "good luck finding where the problem is" state of this error.